go/engine/login_current_device.go

```diff
@@ -34,7 +34,10 @@ func (e *LoginCurrentDevice) Prereqs() Prereqs {
 
 // RequiredUIs returns the required UIs.
 func (e *LoginCurrentDevice) RequiredUIs() []libkb.UIKind {
-	return []libkb.UIKind{}
+	return []libkb.UIKind{
+		libkb.LoginUIKind,
+		libkb.SecretUIKind,
+	}
```

Comment:
They were missing and while writing a test I noticed it as the test didn't provide them to the engine the first time causing a crash.